account_move_line_stock_info/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Move Line Stock Move",
```

Comment:
@bjeficent I think the README title is wrong due to this

Reply:
README updated